src/mergerepo_c.c

```diff
@@ -710,9 +710,10 @@ merge_repos(GHashTable *merged,
 
         g_debug("Processing: %s", repopath);
 
-        if (cr_metadata_load_xml(metadata, ml, NULL) != CRE_OK) {
+        if (cr_metadata_load_xml(metadata, ml, &err) != CRE_OK) {
```

Comment:
Right now this will cause a problem when doing a build without libmodulemd because `err` is defined only when `WITH_LIBMODULEMD` is used.
I think we should move the`err` definition out of the `ifdef WITH_LIBMODULEMD` scope.

Reply:
Done